athena-gcs/README.md

```diff
@@ -0,0 +1,121 @@
+* **Composite Handler:** A Lambda handler retrieves metadata and data records from your database instance(s). This is recommended to be set as a Lambda function handler.
+
+### Database-specific handler parameters
+
+Database-specific metadata (extracted from GCS buckets and files inside those buckets) be used to list buckets (hence database) and tables (hence tables) and record handlers is used to fetch records and spill the data to an S3 bucket.
+
+|Handler|Class|
+|---|---|
+|Composite Handler|GCSCompositeHandler|
+|Metadata Handler|GCSMetadataHandler|
+|Record Handler|GCSRecordHandler|
+
+### Spill parameters:
+
+Lambda SDK may spill data to S3. All database instances are accessed using a single Lambda spill to the exact location.
+
+```
+spill_bucket  Spill bucket name. Required.
+spill_prefix  Spill bucket key prefix. Required.
+```
+
+### GCS Data types supported
+
+| PARQUET       | Arrow     |
+|-----------|-----------|
+| BINARY | VARCHAR|
+| BOOLEAN |BOOL|
+| DOUBLE| DOUBLE|
+| ENUM| VARCHAR|
+| FIXED_LEN_BYTE_ARRAY| DECIMAL|
+| FLOAT | FLOAT (32-bit)|
+| INT32|INT32  or DATEDAY (When Parquet’s column’s logical type is DATE)|
+| INT64 | INT64 OR Timestamp|
+| INT96 | Timestamp|
+| MAP | VARCHAR|
+| LIST | VARCHAR|
+
+**NOTE:** For CSV, all types are converted as Apache Arrow’s VARCHAR type by default
+
+### Secrets
+
+Secret Name and Credential Kay Name are two essential configuration elements without which it is impossible for the connector to list buckets and files as databases and tables using the Google GCS SDK.
+
+Following are two such configurations:
+
+* **GcsSecretName** Name of the secret in the AWS Secret Manager
+* **GcsCredentialKey** Name of the key that contains GCS credential JSON (generated via service account key)
+
+Usually, a JSON set against the **GcsCredentialKey** value that looks like the following:
+
+```json
+{
+ "type": "service_account",
+ "project_id": "[Your GCP project id]",
+ "private_key_id": "[Private key id, automatically generated for you]",
+ "private_key": "-----BEGIN PRIVATE KEY-----\n[The entire private key string automatically inserted during generation]\n-----END PRIVATE KEY-----\n",
+ "client_email": "[Auto generated email for the specified IAM user]",
+ "client_id": "[Id of the client based on IAM user]",
+ "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+ "token_uri": "https://oauth2.googleapis.com/token",
+ "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+ "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/[auto-inserted id for the IAM user]"
+}
+```
+
+### Deploying The Connector
+
+To use the Amazon Athena GCS Connector in your queries, build and deploy this connector from the source; follow the below steps:
+
+1. From the athena-federation dir, run `mvn clean install` if you haven't already.
+2. From the athena-storage-api dir, run `mvn clean install` if you haven't already.

```

Comment:
I don't see this dir, did we remove it or not include in this PR

Reply:
we will be sharing the readme file separately by mail.